pkg/vm/engine/tae/logtail/service/session.go

```diff
@@ -404,7 +404,7 @@ func (ss *Session) SendErrorResponse(
 func (ss *Session) SendSubscriptionResponse(
 	sendCtx context.Context, tail logtail.TableLogtail,
 ) error {
-	ss.logger.Info("send subscription response", zap.Any("table", tail.Table), zap.String("To", tail.Ts.String()))
+	ss.logger.Debug("send subscription response", zap.Any("table", tail.Table), zap.String("To", tail.Ts.String()))
```

Comment:
low frequency, suggest keeping it as the original. subsciption is only frequent in BVT scenario